nat/mapping/port_mapping.go

```diff
@@ -18,10 +18,12 @@
 package mapping
 
 import (
+	"fmt"
 	"time"
 
 	log "github.com/cihub/seelog"
 	portmap "github.com/ethereum/go-ethereum/p2p/nat"
+	"github.com/mysteriumnetwork/node/metrics"
```

Comment:
Lets not couple low level packages with application logic it self. How could we achieve that?

Reply:
By using `EventBus`, which @zolia is working on :)